extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -23,6 +22,15 @@ const CtaAccentElement = {
   BACKGROUND: 'background',
 };
 
+/**
+ * @enum {string}
+ */
+export const AttachmentTheme = {
```

Comment:
Duplicating enums is a big no, no one assumes there are multiple and they get confusing/out of sync

Reply:
I moved it from the other file to this one because the other file installs all the custom elements, so if I import anything from it into here, it would bundle all the code in `amp-story.js`. The way to prevent this is to not have any dependencies on `amp-story-page-attachment.js`, otherwise it imports everything here.